lib/css_parser/rule_set.rb

```diff
@@ -70,15 +93,8 @@ def add_declaration!(property, value)
         return
       end
 
-      value = value.gsub(/;\Z/, '')
-      is_important = !value.match(CssParser::IMPORTANT_IN_PROPERTY_RX).nil?
-      value = value.gsub(CssParser::IMPORTANT_IN_PROPERTY_RX, '')
-      property = property.downcase
-      property.strip!
-      # puts "SAVING #{property}  #{value} #{is_important.inspect}"
-      @declarations[property] = {
-        value: value, is_important: is_important, order: @order += 1
-      }
+      property = property.downcase.strip
```

Comment:
avoids 1 extra string:
```suggestion
      property = property.downcase
      property.strip!
```